packages/@ama-sdk/showcase-sdk/readme.md

```diff
@@ -1,7 +1,7 @@
 # Showcase SDK
 
->Note:
-This package is intended for **testing purposes only**. It is not suitable for production use.
+> [!CAUTION]
+> This package is intended for **testing purposes only**. It is not suitable for production use.
```

Comment:
As we added the 'deprecate' in package.json, the output looks like:
![image](https://github.com/AmadeusITGroup/otter/assets/110401537/71893a31-548c-477e-8352-b33eea305805)

Should not be too much to have the [Caution], too ?
Or maybe remove the **deprecate** and keep only the **caution** ?